lambda/src/components/SelectedNote.js

```diff
@@ -0,0 +1,154 @@
+import React from "react";
+import axios from "axios";
+import SideView from "./SideView";
+// import NotesForm from "./NotesForm";
+// import CreateNote from "./CreateNote";
+
+class SelectedNote extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      note: "",
+      toggleUpdate: true,
+      toggleDelete: true,
+      toggleLinks: true,
+      updateTitle: "",
+      updateContent: ""
+    };
+  }
+
+  componentDidMount() {
+    const noteId = this.props.match.params.id;
+    this.getNote(noteId);
+  }
+
+  getNote = noteId => {
+    axios
+      .get(`https://killer-notes.herokuapp.com/note/get/${noteId}`)
+      .then(response => {
+        console.log("HELLO FROM GET - RESPONSE IS: ", response.data);
```

Comment:
It is a best practice to remove these types of 'testing' console.logs from your code before your final commit/pull request.